README.md

```diff
@@ -19,3 +19,25 @@ Before installing these plugins be sure that you uninstall their old standalone
 packages, like `pip3 uninstall molecule-azure`. If you fail to do so, you will
 end-up with a broken setup, as multiple plugins will have the same entry points,
 registered.
+
+## Creating new releases
+
+The `release.yml` workflow that generates the wheel and uploads the release to PyPI.
+Here are the steps you need to kick that process off:
+
+1. Review the commit logs and decide on the next version.
+   - Breaking changes should increment to a new major version.
+   - New features should increment to a new minor version.
+   - Bug fixes and small changes should increment to a new patch version.
+
+2. Create a new tag and push it to the repo.
+
+   ```bash
+   git tag <NEW_VERSION>
+   git push --tags upstream
+   ```
+
```

Comment:
The GitHub docs below talks through adding a tag during the release. Is it worth trimming this down to just use the GitHub docs to describe the full release steps?

Reply:
sounds reasonable.